tests/unit-tests/datasets/postfix_formatting/index.rst

```diff
@@ -0,0 +1,8 @@
+:orphan:
+
+Index page
+==========
+
+.. toctree::
+    readme
```

Comment:
I assume there should be a newline here between the directive definition and its contents.